src/store/api.ts

```diff
@@ -242,11 +246,21 @@ export const getDepositorInfoAccountData = multiAsync(
 );
 
 export const createDepositor = multiAsync(async (connection: Connection, wallet: Wallet) => {
```

Comment:
This fails when I try to run the app and deposit funds

Reply:
In the tests, you do: 
```
const [signingAuthorityPda, _] = await anchor.web3.PublicKey.findProgramAddress(
      [Buffer.from(program.programId.toString()).subarray(0, 32)],
      program.programId
    );
```
In the app you do: 
```
const findSigningAuthorityPDA = multiAsync(async () => {
	const seeds: PdaSeeds = [config.clusterConfig.programId.toBuffer()];
	return findPDA(seeds);
});
```
=> are those the same?

Reply:
no, we need the subarray version. I'll fix